examples/src/main/java/ai/djl/examples/inference/benchmark/util/AbstractBenchmark.java

```diff
@@ -287,7 +288,10 @@ public NDList processInput(TranslatorContext ctx, Object input) {
                         /** {@inheritDoc} */
                         @Override
                         public Object processOutput(TranslatorContext ctx, NDList list) {
-                            return list.get(0).toFloatArray();
```

Comment:
Does the call to toFloatArray() not work?

Reply:
The output dataType may not be float32.

Reply:
Then, you could to `.toType(..).toFloatArray()` or `.toArray()`

Reply:
Ort doesn't support toType().